cirq/google/engine/client/quantum_v1alpha1/gapic/quantum_engine_service_client.py

```diff
@@ -179,16 +184,18 @@ def __init__(self,
         # These are the actual callables which invoke the proper
         # transport methods, wrapped with `wrap_method` to add retry,
         # timeout, and the like.
-        self._inner_api_calls = {}
+        self._inner_api_calls: Dict = {}
 
     # Service calls
-    def create_quantum_program(self,
-                               parent=None,
-                               quantum_program=None,
-                               overwrite_existing_source_code=None,
-                               retry=google.api_core.gapic_v1.method.DEFAULT,
-                               timeout=google.api_core.gapic_v1.method.DEFAULT,
-                               metadata=None):
+    def create_quantum_program(
+            self,
+            parent: str = None,
```

Comment:
`None` is definitely not an instance of the `str` type, so this should be `Optional[str]`.